contributors/index.js

```diff
@@ -1383,5 +1383,12 @@ module.exports.list = [
     role: 'Student',
     tech_stack: ['Python', 'C#', 'Objective C', 'Swift'],
     github_username: 'daarcher04'
+  },
+ {
```

Comment:
There is a slight padding misalignment here. It's very minor but since we have a prettier config right in the project , It's kinda becomes redundant if we don't follow the style guidelines. Are you not using prettier extension ? Generally these type of things are auto formatted by the extension.